extra/gitdm/company-map

```diff
@@ -1,6 +1,8 @@
 northern.tech Northern.tech
 mender.io Northern.tech
 cfengine.com Northern.tech
+tranchitella.eu Northern.tech
+users.noreply.github.com Northern.tech
```

Comment:
Github uses this domain for all people who enabled private email settings. We cannot assume that such emails belong to Northern.tech, particularly when using the Github UI to make changes. There are already [several](https://github.com/mendersoftware/mender/commit/1198c9f34c98a7f16ea020276e7cad8726dabc00) [examples](https://github.com/mendersoftware/mender-docs/commit/23eab8336f6f4d3fa4074ee5d9046d945c936f46) [of this](https://github.com/mendersoftware/mender-docs/commit/5560cafbb6a4dc544e500ba772be3750ade6fd29).

We will have to add his dedicated name.

Reply:
Thanks for resubmitting!

I tried with adding it to `mailmap` but missed the `groups-map` part, so it had no effect... But now I know for next time :)